Eastern-Research-Group/csb-rebate-forms-app
# Feature/build out close out form functionality

## Related Issues:
* CSBAPP-5

## Main Changes:
Continuation of #288 – integrates the initial version of the close-out form into the wrapper app, which includes fetching payment request form submission data from the BAP to inject into a brand new close-out form submission.

## Steps To Test:
1. Work with BAP team to get at least one Payment Request form submission into an "Accepted" state, so you can create a new Close-Out form submission. The flow works the same as it does with selected Application form submissions where a "New Close-Out" button is displayed in the user's dashboard in the table below any selected Payment Request form submissions.
